### PR TITLE
Update manage-creation-of-groups.md

### PR DESCRIPTION
**Pull request by msbemba**
added script to revert changes made

### CODE REVIEW

**Comment by mypil** [comment]
@nenonix @JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: Office365-Admin/create-groups/manage-creation-of-groups.md [hunk 1]
```diff
@@ -189,7 +189,38 @@ The last line of the script will display the updated settings:
 
 If in the future you want to change which security group is used, you can rerun the script with the name of the new security group.
 
-If you want to turn off the group creation restriction and again allow all users to create groups, set `$GroupName` to "" and `$AllowGroupCreation` to "True" and rerun the script.
+If you want to turn off the group creation restriction and again allow all users to create groups, Run the below script
```
Comment (JohanFreelancer9) [inline Office365-Admin/create-groups/manage-creation-of-groups.md]:
```suggestion
If you want to turn off the group creation restriction and allow all users to create groups again, run the following script.
```

**Comment by mypil** [comment]
@msbemba - please merge the commit on the copy/edit review done by JohanFreelancer9 or make the necessary changes on the PR following the suggestions made. Thank you.

**Comment by mypil** [comment]
@MikePlumleyMSFT - PR has been copy edited and is ready for final review. This is based on issue #818.

Thank you.

cc: @kenwith

**Comment by MikePlumleyMSFT** [comment]
I'm not sure I follow - @msbemba what is the new script for?

**Comment by msbemba** [comment]
> I'm not sure I follow - @msbemba what is the new script for?

When admins want to unlock users to be able to create groups again, the instructions mentioned in the document says to " set $GroupName to "" and $AllowGroupCreation to "True" and re run the script" however this doesn't help as the GroupCreationAllowedGroupID still has the object ID value of the old group. So the new script has the addition of" Else $settingsCopy["GroupCreationAllowedGroupId"] = $GroupName " which will clear the value and lets users created groups again.

Thanks
Sri

**Comment by MikePlumleyMSFT** [comment]
Thanks, @msbemba. Can we just add the else statement to the original script instead of duplicating the script?

**Comment by msbemba** [comment]
> Thanks, @msbemba. Can we just add the else statement to the original script instead of duplicating the script?

Makes sense, I have made the change as per your suggestion. @MikePlumleyMSFT

**Comment by MikePlumleyMSFT** [comment]
Thanks!

**Pull request merged**